chellow/templates/site_edit.html

```diff
@@ -33,13 +33,8 @@
 					<form method="post" action="">
 						<fieldset>
 							<input type="hidden" name="site_id" value="{{site.id}}">
-							<!-- <legend>
-								Are you sure you want to delete this site and any associated snags?
-							</legend> -->
 							<input type="submit" name="delete" value="delete"> 
-							<input type="reset" value="Cancel" onClick="window.location='/sites/{{site.id}}/edit'" />
-							
-						</fieldset>
+							<input type="reset" value="Cancel" onClick="window.location='{{context_path}}/sites/{{site.id}}/edit'" />													</fieldset>
```

Comment:
Probably not a good idea to use JavaScript for just the cancel button. Better to do with HTML / CSS.